rbac/providers/common/inbound_filters.py

```diff
@@ -68,17 +68,10 @@ def datetime_to_seconds(value):
 def inbound_value_filter(value):
     """Cleans up data values
     1. Unwraps LDAP attributes
-    2. Removes empty arrays
-    3. Unwraps single value arrays
-    4. Converts datetime.datetime to seconds
+    2. Converts datetime.datetime to seconds
```

Comment:
The input parameter should be named "inbound_value". This is to remove the confusion on this line: 
value = value.value